server/controllers/redirectController.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, RequestHandler, Response } from 'express'
+import { Path } from 'static-path'
+
+export default class RedirectController {
+  redirect<T extends string>(path: Path<T>): RequestHandler {
+    return async (_req: Request, res: Response) => {
+      return res.redirect(301, path.pattern)
```

Comment:
I think it would be clearer if we didn't return the result of `res.redirect` as it is `void`